update_prevalence.py

```diff
@@ -1442,16 +1458,14 @@ def process_regional_case_reports():
                 manufacturer: shots / total_administered_shots
                 for (manufacturer, shots) in administered_shots_by_manufacturer.items()
             }
-            for k, v in proportional_weights.items():
+            for manufacturer, proportion in proportional_weights.items():
+                most_recent = provincial_reports.data[-1]
+                total_fully_vaccinated = most_recent.vaccine_coverage_dose_2
+                partially_vaccinated = most_recent.vaccine_coverage_dose_1 - most_recent.vaccine_coverage_dose_2
                 place.set_vaccines_of_type(
-                    k,
-                    v
-                    * get_partially_vaccinated(
-                        provincial_reports.summary[-1].cumulative_avaccine,
-                        provincial_reports.summary[-1].cumulative_cvaccine,
-                        2,
-                    ),
-                    v * provincial_reports.summary[-1].cumulative_cvaccine,
+                    manufacturer,
+                    round(proportion * partially_vaccinated),
+                    round(proportion * total_fully_vaccinated),
```

Comment:
The current API data gives us the number of people who have gotten one, two, three, four, etc shots.

I don't think it'd be correct to pass that information into this function:

```python
    def get_partially_vaccinated(total_shots, total_fully_vaccinated, shots_for_full_vaccination) -> int:
        return total_shots - shots_for_full_vaccination * total_fully_vaccinated
```

The second parameter is meant to be the total number of shots administered in the area.  Perhaps that's what `cumulative_cvaccine` meant (?), but it's certainly not what `vaccine_coverage_dose_2` means.  I believe that `vaccine_coverage_dose_2` means "the number of people who have gotten two doses of the vaccine.  As such, the logic is simpler:

```python
                total_fully_vaccinated = most_recent.vaccine_coverage_dose_2
                partially_vaccinated = most_recent.vaccine_coverage_dose_1 - most_recent.vaccine_coverage_dose_2
```

Reply:
Note that both vaccine _coverage_ (% of population) and _administration_ (raw dose numbers) are available. Presently, the former is an adaption PHAC vaccine coverage data (ccodwg/CovidTimelineCanada#21) whereas the latter is an adaption of the PHAC vaccine administration data (ccodwg/CovidTimelineCanada#47). The former `cumulative_cvaccine` value is equivalent to `vaccine_administration_dose_2` in the current dataset.

Reply:
@jeanpaulrsoucy: Thanks for the clarification!  Is `cumulative_avaccine` the same as `vaccine_administration_dose_1`?  i.e., the number of people who have gotten at least one shot?

The current microCOVID code assumed that `cumulative_avaccine` counts the total number of shots given, not the number of people who have had at least one shot.

Reply:
@jeanpaulrsoucy : One other question - could you clarify the meaning of `vaccine_administration_total_doses`?  It seems to be the sum of the `vaccine_administration_dose_*` values - but if my assumption is correct about what those fields mean, that wouldn't be a number that would have any meaning.

Reply:
`cumulative_avaccine` is equivalent to `vaccine_administration_total_doses`, which is the sum of all `vaccine_administration_dose_n` values.

Reply:
OK, thanks - I'll dig in on that and double check my assumptions.

Reply:
All right, I'm convinced that the logic as I added is correct.  The old logic was correct enough until boosters started to happen, at which point the computed number of partially vaccinated people started trending under the real value, as each booster shot administered inappropriately reduced it by one.

I believe the other (existing) use of `get_partially_vaccinated()` is fine, as it uses api.covid19tracker.ca's `total_vaccinations` and `total_vaccinated` fields, which seem like what the function was intended to use.

Reply:
This seems right to me; previously cumulative_avaccine was all we could get from the API.